### ISSUE TITLE
Please add ColdFusion / CFML language support

### ISSUE THREAD

**Issue by JamoCA** [kucherenko/jscpd#619]
Hi! Thank you for your work on this tool! Could you please add ColdFusion / CFML language support?

ColdFusion is the official language from Adobe. It's been in use since 1995 (and has been the core language I've used since '97.)
https://helpx.adobe.com/coldfusion/user-guide.html

Lucee CFML is an alternative open-source CFML platform and has some tags & functions that differ slightly from Adobe's.
https://docs.lucee.org/

**Comment by kucherenko** [comment]
thank you, will investigate

**Comment by akashbavlecha** [comment]
Yeah, it will be a great edition. A lot of ColdFusion programmers I know will certainly benefit from this.

**Comment by JamoCA** [comment]
What's required in order for a language to be supported by jscpd?  (Is there any documentation regarding this?)

I see a list of languages here. They all appear to reference the [RePrism](https://github.com/tannerlinsley/reprism) project.
https://github.com/kucherenko/jscpd/blob/master/packages/tokenizer/src/grammar-loader.ts

The RePrism project is a tweaked version of [PrismJS](https://github.com/PrismJS/prism).  A "cfscript" language file (which support script syntax) was merged back in [early 2021](https://github.com/PrismJS/prism/pull/2771).
https://github.com/PrismJS/prism/blob/master/components/prism-cfscript.js

Does the RePrism project (which hasn't push any updates in the last 6 years) need to be requested to include the new cfscript language file?

Thanks.

**Comment by kucherenko** [comment]
published in jscpd@4.1.0
